src/Widgets/ImageWidget/DimensionsPicker.tsx

```diff
@@ -14,7 +14,60 @@ export const DimensionsPicker = connect(function DimensionsPicker({
     <div
       className={`dimensions-picker scrivito_detail_content scrivito_${theme}`}
     >
-      Dimensions picker
+      <div className="row">
+        <div className="col-6">
+          <div className="scrivito_detail_label">
+            <span>Width</span>
+          </div>
+
+          <div className="item_content">
+            <div className="input_group">
+              <input type="text" placeholder="100" />
+
+              <select>
+                <option value="pixel">px</option>
+                <option value="percent">% </option>
+              </select>
+            </div>
+          </div>
+        </div>
+
+        <div className="col-6">
+          <div className="scrivito_detail_label">
+            <span>Height</span>
+          </div>
+
+          <div className="item_content">
+            <div className="input_group">
+              <input type="text" placeholder="100" />
+              <span className="input_group_text">px</span>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div className="scrivito_detail_label">
+        <span>Object fit</span>
+      </div>
+      <div className="item_content">
+        <ul className="enum_attribute">
+          <li
+            className="enum_attribute_active"
+            title="Image is scaled to maintain its aspect ratio while fitting within the element's content box."
+          >
+            <div class="attribute-preview contain"></div>
+            <span>Contain</span>
+          </li>
+          <li title="Image is sized to maintain its aspect ratio while filling the element's entire content box. The Image will be clipped to fit.">
+            <div class="attribute-preview cover"></div>
+            <span>Cover</span>
+          </li>
+          <li title="Image is sized to fill the element's content box. If necessary, the image will be stretched or squished to fit.">
+            <div class="attribute-preview fill"></div>
+            <span>Fill</span>
+          </li>
+        </ul>
```

Comment:
Sorry for the belated response - I just noticed this now.

Please also add `none` and `scale-down` to support all currently know values of "object-fit" - see https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit#try_it